contracts/eosio.system/include/eosio.system/eosio.system.hpp

```diff
@@ -622,8 +622,8 @@ namespace eosiosystem {
    // - `version` defaulted to zero,
    // - `from` account creating and paying for loan,
    // - `receiver` account receiving rented resources,
-   // - `payment` SYS tokens paid for the loan,
-   // - `balance` is the amount of SYS tokens available to be used for loan auto-renewal,
+   // - `payment` EOS tokens paid for the loan,
```

Comment:
Why does this file still keep `EOS`, not use `A` or `Vaulta`?

Reply:
Only `core.vaulta` contract support `A`. system-contracts only support `EOS`.